installation/index.md

```diff
@@ -36,13 +36,16 @@ The previously mentioned [openHABian](openhabian.html) can also be used to kicks
 
 ## Prerequisites
 
-Make sure that you have an up to date version of the [Oracle Java Runtime Environment](https://java.com/en/download/manual.jsp) on your host system installed.
-A famous choice is OpenJDK, however, Oracle Java Version 8 is recommended for openHAB.
-openHAB may be unstable and slower executed by OpenJDK.
+Make sure that you have an up to date Java Runtime Environment (JRE) installed on your host system. The [Oracle JRE](https://java.com/en/download/manual.jsp)
```

Comment:
@kaikreuzer any news?
After messing around in the code and testing a few openHABian installations, I realized, that Oracle Java in the current way of installation is the cause for a few inpatient people *thinking* their installation was not successful... Nothing critical but I could just switch over to Zulu and be done with that. This will also pave the way for the pine 32bit problem.

Did anyone already do a performance comparison?

So I would do the manual installation from tar for now and sneak in the repository whenever it is available. How does that sound for you? :wink: